src/js/actions/ActionsCreators.js

```diff
@@ -280,8 +279,11 @@ export function setPayout(payload) {
     if (typeof payout === 'undefined') {
       payout = getState().getIn(['storage', 'payout']);
     }
+    else if (isNaN(payout*1) || !Number.isInteger(payout*1) || parseInt(payout) < 0 || parseInt(payout) > 100) {
```

Comment:
you can simply do this

payout = +(getState().getIn(['storage', 'payout']));

`+` will do type conversion from watever to number, and return Nan if it's inconvertible

`else if (isNaN(payout) || payout < 0 || payout > 100) `